modelservice/src/main/sql/build-db.sql

```diff
@@ -58,4 +58,18 @@ create table user (
   password     varchar(100) not null,
   role         varchar(100),
   enabled      boolean
-);
\ No newline at end of file
+);
+
+create table metric_tags (
+tag_id        int NOT NULL AUTO_INCREMENT PRIMARY KEY , 
+tag_keys      varchar(255) NOT NULL, 
```

Comment:
@tusharbahl make both key and value unique

Reply:
Done